pheno_images/curate_weather.R

```diff
@@ -24,11 +26,26 @@ for(site in 1:nrow(efi_sites)){
 }
 
 ###########Clean up weather data###########
+#To calculate daily total shortwaves, multiply srad by daylength and convert to Megajoules m^-2 d^-1
 clean_daymet <- all_daymet %>% 
   mutate(origin_year = year - 1, 
          origin_date = paste0(origin_year, "-12-31"), 
          date = as.Date(yday, origin = origin_date)) %>% 
-  select(-year, -yday, -origin_year, -origin_date)
+  select(-year, -yday, -origin_year, -origin_date) %>%
+  rename(siteID = site,
+         max_temp = tmax..deg.c.,
+         min_temp = tmin..deg.c.,
+         precip = prcp..mm.day.) %>%
+  relocate(siteID, date) %>%
+  mutate(radiation = ud.convert(dayl..s.*srad..W.m.2., "joule", "megajoule")) %>%
+         # SVP = esat(max_temp),
+         # vpd = ud.convert(SVP - vp..Pa., "Pa", "kPa"))
+  select(siteID, date, radiation, max_temp, min_temp, precip) %>%
+  arrange(siteID)
+
+#### Write out only weather data file ####
+write.csv(clean_daymet, file = "Daymet_weather.csv", row.names = F)
+
 
 ###########Join weather data to GCC data###########
 pheno_images <- read.csv("pheno_images/targets_gcc.csv") %>% 
```

Comment:
Due to changes in the final `clean_daymet` dataframe, the joining with the GCC dataset needs to change as below. 
```suggestion
pheno_images <- read.csv("pheno_images/targets_gcc.csv") %>% 
  mutate(time = as.Date(time))

gcc_weather <- left_join(pheno_images, clean_daymet, 
                         by = c("siteID" = "siteID", "time" = "date"))

```